test/parallel/test-buffer-readuint8.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUInt8()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUInt8(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+  () => {
+    buf.readUInt8(5);
+  },
+  /Index out of range/
```

Comment:
Nit: how about using `^RangeError: Index out of range$`? Only because this is done in other tests.